src/Functions/Public/iaas/Resize-vRAMachine.ps1

```diff
@@ -97,13 +97,13 @@
 
         $APIUrl = "/iaas/api/machines"
 
-        function CalculateOutput {
+        function CalculateOutput([int]$CompletionTimeout,[switch]$WaitForCompletion,[PSCustomObject]$RestResponse) {
             if ($WaitForCompletion) {
```

Comment:
Although not part of this PR, we should consider using IsPresent() when evaluating switch types as it will give a more reliable result.

Reply:
I went through all of the code and made this change. Whenever a switch is evaluated, it is done so by using the IsPresent function